app/controllers/forgot_passwords_controller.rb

```diff
@@ -0,0 +1,26 @@
+class ForgotPasswordsController < ApplicationController
+
+  def create
+    user = User.where(email: params[:email]).first
```

Comment:
You should use `.find_by` to fetch a single record from database instead of `.where(args).first`. In Rails 3, `.where().first` was an OK way, thats why Kevin uses that in videos, but in Rails 4 `.find_by` is the new standard as its more efficient.